File: Finding area of shapes using different languages/AOS.java
import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner myObj = new Scanner(System.in);
    String shape;
    
    System.out.println("Enter shape(circle,rectangle,triangle):"); 
    shape = myObj.nextLine();   
  
    switch(shape){
        case "circle":
            System.out.println("Enter the radius: ");
            double radius = myObj.nextDouble();
            double area = 3.1416 * radius * radius;
            System.out.println("The area of the circle is " + area);
            break;
        case "rectangle":
            System.out.println("Enter the length: ");
            double length = myObj.nextDouble();
            System.out.println("Enter the width: ");
            double width = myObj.nextDouble();
            double area2 = length * width;
            System.out.println("The area of the rectangle is " + area2);
            break;
        case "triangle":
            System.out.println("Enter the base: ");
            double base = myObj.nextDouble();
            System.out.println("Enter the height: ");
            double height = myObj.nextDouble();
            double area3 = 0.5 * base * height;
            System.out.println("The area of the triangle is " + area3);
            break;
    }
  }
}

File: Finding area of shapes using different languages/AOS.js
var shape = prompt("Enter the shape of your choice (circle, rectangle, triangle)")

if(shape==="circle"){
   var radius = prompt("Enter the radius");
   var area = (3.1457 * (radius**2));
   alert("The area of the circle is "+area);

} else if(shape==="rectangle"){
    var length = prompt("Enter the length");
    var width = prompt("Enter the width");
    var area = (length*width);
    alert("The area of the rectangle is "+area);

} else if(shape==="triangle"){
    var base = prompt("Enter the length of base");
    var height = prompt("Enter the height");
    var area = 0.5*base*height;
    alert("The area of the triangle is "+area);
    
} else{
    alert("Enter a valid shape!!")
}